### PR TITLE
update to latest boot-cljs stuff

### PR DESCRIPTION
**Pull request by martinklepsch**

### CODE REVIEW

**Comment by pandeiro** [comment]
@martinklepsch, I think @piranha already got this stuff in #11, plus a fix for Node.js targets. Have a look.

Now we just need a merge!

File: src/adzerk/boot_reload.clj [hunk 1]
```diff
@@ -74,10 +74,11 @@
     (write-cljs! out (start-server @pod {:ip ip :port port}) on-jsload)
     (comp
       (with-pre-wrap fileset
-        (doseq [f (->> fileset input-files (by-ext [".main.edn"]))]
+        (doseq [f (->> fileset input-files (by-ext [".cljs.edn"]))]
           (let [path     (tmppath f)
                 in-file  (tmpfile f)
                 out-file (io/file tmp path)]
+            (io/make-parents out-file)
```
Comment (martinklepsch) [inline src/adzerk/boot_reload.clj]: I think this is missing in #11 and should be there.

Comment (piranha) [inline src/adzerk/boot_reload.clj]: That's actually not hard to add. Give me a second. :)

**Comment by martinklepsch** [comment]
Oh dang, you're right @pandeiro. Should have checked before..
@piranha It'd be great if you could add that make-parents call to your branch, then I can happily close this.

**Comment by martinklepsch** [comment]
awesome that was fast :)

**Pull request closed without merging**